src/npcmove.cpp

```diff
@@ -3182,8 +3182,8 @@ void npc::do_reload( item &it )
 
     // Note: we may be reloading the magazine inside, not the gun itself
     // Maybe @todo: allow reload functions to understand such reloads instead of const casts
-    auto &target = const_cast<item &>( *reload_opt.target );
-    auto &usable_ammo = reload_opt.ammo;
+    item &target = const_cast<item &>( *reload_opt.target );
```

Comment:
Sure, if that's the style you want then that's the style you want. I was pointing out that the change didn't match the PR description.